exec/java-exec/src/test/java/org/apache/drill/exec/vector/complex/writer/TestComplexTypeReader.java

```diff
@@ -241,4 +252,49 @@ public void testRepeatedJson() throws Exception {
             .go();
   }
 
+  @Test  // DRILL-4410
+  // ListVector allocation
+  public void test_array() throws Exception{
+
+    long numRecords = 100000;
+    String file1 = "/tmp/" + TestComplexTypeReader.class.getName() + "arrays1.json";
```

Comment:
Seems ParquetRecordReaderTest is ignored?

[1] https://github.com/apache/drill/blob/master/exec/java-exec/src/test/java/org/apache/drill/exec/store/parquet/ParquetRecordReaderTest.java#L84